server/src/interfaces/asset.interface.ts

```diff
@@ -138,6 +138,8 @@ export interface AssetFullSyncOptions {
   lastCreationDate?: Date;
   lastId?: string;
   updatedUntil: Date;
+  isArchived?: false;
```

Comment:
Do we need these two options? I think we can drop them and let the server automatically set them: true for owner, false for partner

Reply:
I think the server does set them. This is just the interface for the repository method. It means we can assert the right values are passed in unit tests, not rely on the implementation itself.